android/AndroidManifest.xml

```diff
@@ -52,7 +49,7 @@
             <!-- Background running -->
         </activity>
     </application>
-    <uses-sdk android:minSdkVersion="16" android:targetSdkVersion="28"/>
```

Comment:
I think this is ok. Is 21 the first one that support 64 bit?

Reply:
Seems it first which looses gcc support and starts clang buildings. Min version to build 5.15.